server/schema.js

```diff
@@ -16,19 +13,14 @@ const query = new GraphQLObjectType({
     recentPost: {
       type: BlogPost,
       resolve() {
-        const coll = Collections.posts.rawCollection();
-        return sleep(200)
```

Comment:
@AdamBrodzinski could you also give it a comment? :) I didn't get the delay at first.